tests/lib/mockMagic.py

```diff
@@ -9,3 +11,40 @@ def validate_config_for_device(self, device_mac, config):
             return return_validate_config_for_device['return_validate_intf']
         if config == validate_true['config']:
             return return_validate_config_for_device['return_validate_true']
+
+    def get_image_bundle_by_name(self, name):
+        """
+        mock to get image_bundle
+        """
+        if device_data[0]['imageBundle'] != 'Invalid_bundle_name':
+            return image_bundle
+        else:
+            return None
```

Comment:
not sure wht this has been resolved?

Reply:
so, here I changed it to "name" as per your suggestion as it will have the same values as device_data[0]['imageBundle'] and to return None I am giving the string "Invalid_bundle_name" in input i.e. it is checking for the image which does not exist.